Documentation/Signoffs/detaildesign_camera.md

```diff
@@ -0,0 +1,139 @@
+# Detailed Design for Camera Subsystem within Primary Data Acquisition System
+
+## Function of the Subsystem
+
+The primary data acquisition system will utilize camera sensors to determine the number of cars in the Bell Hall Lot. The cameras will utilize one of two methods for determining the parking count: tracking statically a parking lot sector or tracking dynamically the movement of cars into and out of a sector. These video feeds from these cameras will then be sent to a server where the images will be processed by an artificial intelligence (AI) algorithm to determine the number of cars and thus the number of available Red parking spots in the Bell Hall. This subsystem is a critical subsystem because these cameras will provide the primary data to the system that will determine the count of vehicles in a lot. The cameras is providing this input data to the server.
+
+## Constraints
```

Comment:
Shorten this. There is far too much extraneous text that is not directly about the constraints. Explanations of what is not allowed does not belong here. State the constraint sufficiently and briefly.